File: abc160/a/rust/a.rs
use std::io::Read;

fn coffee(n: &[u8]) -> bool {
    n[2] == n[3] && n[4] == n[5]
}

fn main(){
    let mut buf = String::new();
    std::io::stdin().read_to_string(&mut buf).unwrap();

    let s = buf.split_whitespace().next().unwrap().as_bytes();

    if coffee(s) {
        println!("Yes");
    }
    else {
        println!("No");
    }
}

File: abc160/a/a.c
#include <stdio.h>

int coffee(char *n) {
  if (*(n + 2) == *(n + 3) && *(n + 4) == *(n + 5)) {
    return 1;
  } else {
    return 0;
  }
}

int main(void) {
  char s[10];
  scanf("%s", s);

  if (coffee(s)) {
    printf("Yes\n");
  } else {
    printf("No\n");
  }

  return 0;
}
